docs/organizations/metadata.mdx

```diff
@@ -1,5 +1,92 @@
 ---
-sanity_slug: /docs/organizations/organization-metadata
+title: Organization Metadata
+description: >-
+  Organization objects hold a set of metadata that can be used internally to store arbitrary information.
 ---
 
-# This is a stub
+import { Tables } from "@/components/Table";
+
+# Organization Metadata
+
+**Organization objects hold a set of metadata that can be used internally to store arbitrary information.**
+
+## Overview
+
+Metadata allows for custom data to be saved on the Organization object, Organization Invitation object, and Organization Membership object. There are two types of metadata: "**public**", and "**private**". Choosing which metadata field you should use for setting custom attributes on your organizations becomes a matter of access and visibility.
+
+<Tables
+  headings={["Metadata", "Frontend API", "Backend API"]}
+  headingsMeta={[{maxWidth:'200px'},{maxWidth: '200px'},{maxWidth: '200px'}]}
+  rows={[
+    {
+    cells: [
+      <>private</>, 
+      <>-</>,
+      <>Read/Write</>
+    ],
+    },
+    {
+      cells: [
+        <>public</>,
+        <>Read</>,
+        <>Read/Write</>,
+      ],
+    },
+  ]}
+/>
+
+<Callout type="info" emoji="ℹ️">
+    Please note that Organization Invitations can only include **public** metadata.
+</Callout>
+
+<Callout type="warning">
+    Metadata is limited to **8kb** maximum.
+</Callout>
+
+## Backend metadata
+
+Both public and private metadata are set from the [Backend API](https://clerk.com/docs/reference/backend-api), but public metadata can be accessed from the [Frontend API](https://reference.clerk.dev/reference/frontend-api-reference) and [Backend API](https://clerk.com/docs/reference/backend-api). These types of metadata can be used to access non-sensitive information about an organization, such as their profile picture or their name. They can be used to build UIs where an organization might not be signed in, but you still want to show some of their profile info.
+
+## Organization Metadata
+
+### Public metadata
+
+The **publicMetadata** property should be used if you need to set some metadata from your back-end and have them displayed as read-only on the front-end.
```

Comment:
This is going to be implemented moving forward, but we will use backend and frontend without dashes.

```suggestion
The `publicMetadata` property should be used if you need to set some metadata from your backend and have them displayed as read-only on the frontend.
```